CourseSearchAPIHttpTrigger/tests/test_helper.py

```diff
@@ -0,0 +1,107 @@
+import unittest
+import os
+import sys
+import inspect
+
+CURRENTDIR = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+PARENTDIR = os.path.dirname(CURRENTDIR)
+sys.path.insert(0, PARENTDIR)
+
+from helper import remove_conjunctions
+
+class TestRemoveConjunctions(unittest.TestCase):
+    def test_when_searchable_field_is_empty_string(self):
+        
+        output_result = remove_conjunctions("")
+        self.assertEqual("", output_result)
+
+    def test_when_searchable_field_has_ampersand_conjunction(self):
+        input_field = "Mathematics & Physics"
+        expected_result = "Mathematics Physics"
+        
```

Comment:
Trailing whitespace here and in several other places in this module. Black will fix.